app/controllers/api/internal/v1/laa_references_controller.rb

```diff
@@ -16,6 +15,16 @@ def create
 
       private
 
+        def contract
+          @contract ||= NewLaaReferenceContract.new.call(**transformed_params)
+        end
+
+        def create_laa_reference
+          LaaReferenceCreator.call(defendant_id: transformed_params[:defendant_id],
+                                   user_name: transformed_params[:user_name],
+                                   maat_reference: transformed_params[:maat_reference])
+        end
+
         def create_params
           params.from_jsonapi.require(:laa_reference).permit(allowed_params)
```

Comment:
Would it be worth standardising the require/permit statements? They alter in format between this and the hearings controller